style_editor.html

```diff
@@ -9580,6 +9580,23 @@
   button.style.backgroundColor = framesPerUpdate == 60 ? "dodgerblue" : "";
 }
 
+</script>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/2.0.2/FileSaver.min.js"></script>
```

Comment:
This won't work off-line, and we can't make it work by making it a PWA either.

Reply:
Ok how about now?

Reply:
This works well.